### ISSUE TITLE
User-reported unhelpful error message

### ISSUE THREAD

**Issue by codeworld-feedback** [google/codeworld#1512]
**Program:** https://www.code.world/#PGGvb5B556mlkELoKno0nYg

**Message:**

    program.hs:1:1-8: error:
        Parse error: An equation was expected here.
        Are you missing the equal sign?

**Comment by cdsmith** [comment]
I could reword this to explain that the top level of a program should be equations.

**Comment by cdsmith** [comment]
The reason for extra care here is that this is an especially likely error for newcomers to the language.
